disdrodb/tests/test_io.py

```diff
@@ -270,16 +271,20 @@ def test_get_raw_file_list():
 path_file_temp = os.path.join(PATH_TEST_FOLDERS_FILES, "test_folders_files_creation", folder_name)
 
 
-def test__create_directory():
-    io._create_directory(path_file_temp)
-    if os.path.exists(path_file_temp):
+def test__create_directory(tmp_path):

```

Comment:
Maybe at a given point ... you could search for double `__` across the entire project and replace with single `_`